obspy/core/event/base.py

```diff
@@ -1166,16 +1166,9 @@ class DataUsed(__DataUsed):
     moment-tensor inversion.
 
     :type wave_type: str or :class:`~obspy.core.event.header.DataUsedWaveType`
```

Comment:
Yeah, so maybe only `:type ...: str` and just link to the enum docs in the `:param ..:` verbose description part as done in the last commit by @claudiodsf ?